DolbyIO/Source/Public/DolbyIOSubsystem.h

```diff
@@ -242,6 +254,48 @@ class DOLBYIO_API UDolbyIOSubsystem : public UGameInstanceSubsystem
 	void SetLogSettings(EDolbyIOLogLevel SdkLogLevel = EDolbyIOLogLevel::Info,
 	                    EDolbyIOLogLevel MediaLogLevel = EDolbyIOLogLevel::Off, const FString& LogDirectory = "");
 
+	/** Gets a list of all avialable audio devices. Both input and output.
+	 *
+	 *  Triggers OnAudioDevicesReceived event when ready.
+	 */
+	UFUNCTION(BlueprintCallable, Category = "Dolby.io Comms")
+	void GetAudioDevices();
+
+	/** Gets the current audio input device.
+	 *
+	 *  Triggers OnCurrentAudioInputDeviceReceived event when ready.
+	 */
+	UFUNCTION(BlueprintCallable, Category = "Dolby.io Comms")
+	void GetCurrentAudioInputDevice();
+
+	/** Gets the current audio output device.
+	 *
+	 *  Triggers OnCurrentAudioOutputDeviceReceived event when ready.
+	 */
+	UFUNCTION(BlueprintCallable, Category = "Dolby.io Comms")
+	void GetCurrentAudioOutputDevice();
+
+	/** Sets the preferred audio input device.
+	 *
+	 *  Triggers OnCurrentAudioDeviceChanged event when the change has been made.
+	 */
+	UFUNCTION(BlueprintCallable, Category = "Dolby.io Comms")
+	void SetPreferredAudioInputDevice(const FString& NativeId);
```

Comment:
I would like to remove the word "preferred" from both set functions. I know the reasoning is to follow C++ SDK naming convention here but Unity already breaks it with https://api-references.dolby.io/comms-sdk-dotnet/documentation/unity/visualscripting/nodes.html#set-audio-input-device and I simply believe this word is unnecessary.

Reply:
removed